pkg/ansiescape/split.go

```diff
@@ -0,0 +1,89 @@
+package ansiescape
+
+import "bytes"
+
+// dropCR drops a terminal \r from the data.
+func dropCR(data []byte) []byte {
```

Comment:
The code I was referencing in the golang codebase did not use trim either, I was not sure if using trim would cause a copy of the slice. For reference http://golang.org/src/bufio/scan.go?s=9746:9824#L277